### PR TITLE
Bids 2567/stats page for validators without index

### PR DESCRIPTION
**Pull request by D13ce**
This PR mainly hides the stats tables on the validator page whenever a validator that is still only "deposited" or "pending" is shown.

Furthermore it improves and unifies the error logging within `validator.go`, removes a sentence that makes so sense on `validatornotfound.html` and adds periods to all "not found"-pages.

### CODE REVIEW

File: handlers/validator.go [hunk 1]
```diff
@@ -1805,19 +1857,23 @@ func ValidatorStatsTable(w http.ResponseWriter, r *http.Request) {
...
+		"route": r.URL.String()}
+
 	data := InitPageData(w, r, "validators", "/validators", "", templateFiles)
 
 	// Request came with a hash
 	if utils.IsHash(vars["index"]) {
 		pubKey, err := hex.DecodeString(strings.TrimPrefix(vars["index"], "0x"))
 		if err != nil {
-			logger.Errorf("error parsing validator public key %v: %v", vars["index"], err)
+			utils.LogError(err, "error decoding validator index", 0, errFields)
```
[the remark this answers is not in the document]
Comment (D13ce) [reply handlers/validator.go:1869]: Good point.

File: handlers/validator.go [hunk 2]
```diff
@@ -1805,19 +1857,23 @@ func ValidatorStatsTable(w http.ResponseWriter, r *http.Request) {
...
 			validatorNotFound(data, w, r, vars, "/stats")
 			return
 		}
 		index, err = db.GetValidatorIndex(pubKey)
 		if err != nil {
-			logger.Warnf("error parsing validator pubkey: %v", err)
+			errFields["pubkey"] = pubKey
+			utils.LogError(err, "error getting validator index from db", 0, errFields)
```
[the remark this answers is not in the document]
Comment (D13ce) [reply handlers/validator.go:1876]: Fixed.

File: handlers/validator.go
[the remark this answers is not in the document]
Comment (D13ce) [reply handlers/validator.go:1]: Done.

**Review by Eisei24 — APPROVED** [review]
lgtm

**Pull request merged**